res/controllers/AKAI-MPKmini2-scripts.js

```diff
@@ -0,0 +1,164 @@
+
+var MPKmini = {};
+
+MPKmini.init = function() {
+    this.currentUnitNum = 1;
+    this.currentEffectNum = 1;
+    this.currentFlash = true;
+    this.timerId = 0;
+
+    MPKmini.updateLed();
+    this.timerId = engine.beginTimer(200, "MPKmini.flashLed()");
+};
+
+MPKmini.shutdown = function() {
+    if (this.timerId !== 0)
+        engine.stopTimer(this.timerId);
+    else
+        print("Shutdown: no timer to stop.");
```

Comment:
Please always use braces.

```suggestion
    if (this.timerId !== 0) {
        engine.stopTimer(this.timerId);
    }
```